addons/stock/models/stock_move.py

```diff
@@ -802,12 +802,15 @@ def _merge_moves_fields(self):
 
     @api.model
     def _prepare_merge_moves_distinct_fields(self):
-        return [
+        fields = [
```

Comment:
I think it would be better if you could use another here. `fields` is already taken, called at the beginning with "from odoo import fields" 🧐